<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>clients_report</class>
 <widget class="QWidget" name="clients_report">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>550</width>
    <height>700</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>clients_report</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QWidget {
    background-color: rgb(242, 244, 255); /* Светлый фон */
}
QLineEdit {
    background-color: rgb(255, 255, 255); /* Белый фон для текстового поля */
    border: 2px solid rgb(40, 102, 110); /* Бирюзовая обводка */
    border-radius: 5px; /* Закругленные углы */
    padding: 5px 10px; /* Отступы для удобства ввода */
    font-size: 14px; /* Размер шрифта */
    color: rgb(3, 63, 99); /* Цвет текста */
}

QLineEdit:focus {
    border-color: rgb(3, 63, 99); /* Изменение цвета рамки при фокусе */
}
QComboBox {
    background-color: rgb(255, 255, 255); /* Белый фон для комбобокса */
    border: 2px solid rgb(40, 102, 110); /* Бирюзовая обводка */
    border-radius: 5px; /* Закругленные углы */
    padding: 5px 10px; /* Отступы */
    font-size: 14px; /* Размер шрифта */
    color: rgb(3, 63, 99); /* Цвет текста */
}

QComboBox QAbstractItemView {
    border: 2px solid rgb(40, 102, 110); /* Рамка для выпадающего списка */
    background-color: rgb(255, 255, 255); /* Белый фон */
    color: rgb(3, 63, 99); /* Цвет текста */
}
QLabel {
    color: rgb(3, 63, 99); /* Темный синий цвет текста */
    font-size: 18px; /* Размер шрифта */
	border-bottom: 0
}
QMessageBox {
    background-color: rgb(242, 244, 255); /* Светлый фон */
    color: rgb(3, 63, 99); /* Темно-синий цвет текста для контраста */
    border: 2px solid rgb(40, 102, 110); /* Бирюзовая обводка */
    border-radius: 8px; /* Закругленные углы */
}

QMessageBox QLabel {
    color: rgb(3, 63, 99); /* Темный цвет текста */
    font-size: 14px; /* Размер шрифта */
}
QMessageBox QPushButton {
    background-color: rgb(3, 63, 99); /* Синий фон кнопки */
    color: rgb(252, 247, 255); /* Белый цвет текста */
    border: none;
    border-radius: 5px;
    padding: 6px 12px; /* Отступы для приятного вида */
}
QMessageBox QPushButton:hover {
    background-color: rgb(40, 102, 110); /* Цвет при наведении */
}
QInputDialog {
    background-color: rgb(242, 244, 255); /* Светлый фон */
    color: rgb(3, 63, 99); /* Темно-синий цвет текста */
    border: 2px solid rgb(40, 102, 110); /* Бирюзовая обводка */
    border-radius: 8px; /* Закругленные углы */
}

QInputDialog QLabel {
    color: rgb(3, 63, 99); /* Темный синий цвет текста */
    font-size: 14px; /* Размер шрифта */
}

QInputDialog QLineEdit {
    background-color: rgb(252, 247, 255); /* Светлый фон поля ввода */
    color: rgb(3, 63, 99); /* Темный текст */
    border: 1px solid rgb(124, 152, 133); /* Граница поля ввода */
    border-radius: 5px; /* Закругленные углы поля ввода */
    padding: 4px; /* Внутренние отступы */
}

QInputDialog QPushButton {
    background-color: rgb(3, 63, 99); /* Синий фон кнопки */
    color: rgb(252, 247, 255); /* Белый цвет текста */
    border: none;
    border-radius: 5px;
    padding: 6px 12px; /* Отступы для приятного вида */
}

QInputDialog QPushButton:hover {
    background-color: rgb(40, 102, 110); /* Цвет при наведении */
}
QPushButton {
    background-color: rgb(3, 63, 99); /* Тёмно-синий основной цвет */
    color: rgb(252, 247, 255); /* Белый цвет текста */
    border: 2px solid rgb(40, 102, 110); /* Бирюзовая обводка */
    border-radius: 10px; /* Закруглённые края */
    padding: 8px 16px; /* Отступы для приятного вида */
    font-size: 14px; /* Размер шрифта */
    font-weight: bold; /* Жирный текст */
}
QPushButton:hover {
    background-color: rgb(40, 102, 110); /* Цвет кнопки при наведении */
    color: rgb(218, 223, 247); /* Голубой текст при наведении */
}
QPushButton:pressed {
    background-color: rgb(124, 152, 133); /* Цвет при нажатии */
}
QScrollArea {
    background-color: rgb(242, 244, 255); /* Светлый фон */
    border: 1px solid rgb(169, 211, 255); /* Тонкая граница */
    border-radius: 5px; /* Закругленные углы */
}
QScrollBar:vertical {
    background-color: rgb(242, 244, 255); /* Цвет фона скроллбара */
    width: 10px; /* Ширина скроллбара */
}
QScrollBar::handle:vertical {
    background-color: rgb(3, 63, 99); /* Основной цвет ползунка */
    border-radius: 5px;
    min-height: 20px; /* Минимальная высота ползунка */
}
QScrollBar::add-line:vertical, QScrollBar::sub-line:vertical {
    background: none; /* Без линий сверху и снизу */
    height: 0px;
}
</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_3">
   <item row="0" column="0">
    <layout class="QVBoxLayout" name="verticalLayout_2">
     <item>
      <layout class="QGridLayout" name="gridLayout_2">
       <item row="0" column="3">
        <spacer name="horizontalSpacer_6">
         <property name="orientation">
          <enum>Qt::Orientation::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item row="0" column="0">
        <spacer name="horizontalSpacer_5">
         <property name="orientation">
          <enum>Qt::Orientation::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item row="0" column="1">
        <widget class="QLabel" name="label_4">
         <property name="styleSheet">
          <string notr="true">QLabel {
	font-size:25px;
}</string>
         </property>
         <property name="text">
          <string>Client filters:</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QFormLayout" name="formLayout">
       <item row="0" column="0">
        <widget class="QLabel" name="label_3">
         <property name="text">
          <string>Name filter:</string>
         </property>
        </widget>
       </item>
       <item row="0" column="1">
        <widget class="QLineEdit" name="nameLineEdit">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="placeholderText">
          <string>ex. Irina</string>
         </property>
        </widget>
       </item>
       <item row="1" column="0">
        <widget class="QLabel" name="label_2">
         <property name="text">
          <string>Username filter:</string>
         </property>
        </widget>
       </item>
       <item row="1" column="1">
        <widget class="QLineEdit" name="usernameLineEdit">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="placeholderText">
          <string>ex. IrishBee</string>
         </property>
        </widget>
       </item>
       <item row="2" column="0">
        <widget class="QLabel" name="label">
         <property name="text">
          <string>Social status filter: </string>
         </property>
        </widget>
       </item>
       <item row="2" column="1">
        <widget class="QComboBox" name="socialStatusComboBox">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <spacer name="horizontalSpacer">
         <property name="orientation">
          <enum>Qt::Orientation::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <widget class="QPushButton" name="generateButton">
         <property name="text">
          <string>Generate report</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="saveButton">
         <property name="text">
          <string>Save report</string>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="horizontalSpacer_2">
         <property name="orientation">
          <enum>Qt::Orientation::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_3">
       <item>
        <spacer name="horizontalSpacer_3">
         <property name="orientation">
          <enum>Qt::Orientation::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <widget class="QPushButton" name="backButton">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="text">
          <string>Back</string>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="horizontalSpacer_4">
         <property name="orientation">
          <enum>Qt::Orientation::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
